src/python/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+find_package(Python COMPONENTS Interpreter Development)
```

Comment:
should we check the python version ?

Reply:
No, it generates bindings for any Python version.